static/sass/components/_author.scss

```diff
@@ -0,0 +1,108 @@
+// Author Box
+// ==========
+
+[data-module] {
+  + [data-module='rst-author'] {
```

Comment:
OK, tried to remove the parent and using only [data-module='rst-author']  wasn't enough. Am I doing something wrong?

![screen shot 2017-02-08 at 1 07 27 pm](https://cloud.githubusercontent.com/assets/1581694/22752905/c8cb0ebe-edff-11e6-90a8-846101c0d581.jpg)